wavelength-plugins/ssb.js

```diff
@@ -578,40 +577,32 @@ class SSB {
 	}
 }
 
+const ssbData = {};
+
 //Load JSON
 try {
-	fs.accessSync("config/ssb.json", fs.F_OK);
-} catch (e) {
-	fs.writeFile("config/ssb.json", "{}", function (err) {
-		if (err) {
-			console.error(`Error while loading SSBFFA: ${err}`);
-			ssbWrite = false;
-		} else {
-			console.log("config/ssb.json not found, creating a new one...");
-		}
-	});
-	noRead = true;
+	ssbData = FS("config/ssb.json").readIfExistsSync();
```

Comment:
There is no way you tested this.

Your trying to write to a `const` variable, that's a syntax error on server startup.